deploy/sre-prometheus/100-managed-upgrade-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-upgrade-operator-alerts
+    role: alert-rules
+  name: sre-managed-upgrade-operator-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-upgrade-operator-alerts
+    rules:
+    - alert: UpgradeConfigValidationFailed
+      # Get alert immediately once the upgradeconfig validation failed
+      expr: |
+        upgradeoperator_upgradeconfig_validation_failed > 0
+      for: 5m
+      labels:
+        severity: critical
+        namespace: openshift-monitoring
+      annotations:
+        summary: "Upgrade config validation failed"
+        description: "Upgrade config validation failed"
+    - alert: ClusterCheckFailed
+      # Get alert immediately once the cluster check failed
+      expr: |
+        upgradeoperator_cluster_check_failed > 0
+      for: 5m
+      labels:
+        severity: critical
+        namespace: openshift-monitoring
+      annotations:
+        summary: "cluster check failed"
+        description: "basic cluster checks failed on either before the upgrade or after the upgrade"
+    - alert: NodeScalingFailed
```

Comment:
Again maybe some additional context to the alert name here to indicate it's scaling due to a cluster upgrade. eg "UpgradeNodeScalingFailedSRE"